I still get an error on this:
```
$ nix --experimental-features nix-command build --no-link --keep-going --option build-use-sandbox relaxed -f /home/pinpox/.cache/nixpkgs-review/pr-118712/build.nix
error: builder for '/nix/store/2hw8yj4vljn7s46nrsxim3i49hc0zja7-wezterm-20210407-nightly-vendor.tar.gz.drv' failed with exit code 1;
       last 10 log lines:
       > Caused by:
       >   feature `resolver` is required
       >
       >   consider adding `cargo-features = ["resolver"]` to the manifest
       > Traceback (most recent call last):
       >   File "/nix/store/4pvjb8h2cyzxbqc4angl6f4g9jnx1qxw-cargo-vendor-normalise/bin/.cargo-vendor-normalise-wrapped", line 42, in <module>
       >     main()
       >   File "/nix/store/4pvjb8h2cyzxbqc4angl6f4g9jnx1qxw-cargo-vendor-normalise/bin/.cargo-vendor-normalise-wrapped", line 17, in main
       >     assert list(data.keys()) == ["source"]
       > AssertionError
       For full logs, run 'nix log /nix/store/2hw8yj4vljn7s46nrsxim3i49hc0zja7-wezterm-20210407-nightly-vendor.tar.gz.drv'.
error: 1 dependencies of derivation '/nix/store/kk9hpwg1yi9qahwxb5ppw238g7nhv0yd-wezterm-20210407-nightly.drv' failed to build
error: 1 dependencies of derivation '/nix/store/crdw3ihn4a426w1g286dgk6lv5x18mx7-env.drv' failed to build
```

--- a/pkgs/applications/terminal-emulators/wezterm/default.nix
+++ b/pkgs/applications/terminal-emulators/wezterm/default.nix
@@ -13,11 +13,13 @@
 , CoreGraphics
 , Cocoa
 , Foundation
+, libiconv
 
 , dbus
 , libX11
 , xcbutil
 , libxcb
+, xcbutilimage
 , xcbutilkeysyms
 , xcbutilwm # contains xcb-ewmh among others
 , libxkbcommon
@@ -38,6 +40,7 @@ let
     libX11
     xcbutil
     libxcb
+    xcbutilimage
     xcbutilkeysyms
     xcbutilwm
     libxkbcommon
@@ -51,22 +54,23 @@ let
     Foundation
     CoreGraphics
     Cocoa
+    libiconv
   ];
   pname = "wezterm";
 in
 
 rustPlatform.buildRustPackage {
   inherit pname;
-  version = "unstable-2020-11-22";
+  version = "20210314";
 
   src = fetchFromGitHub {
     owner = "wez";
     repo = pname;
-    rev = "3bd8d8c84591f4d015ff9a47ddb478e55c231fda";
-    sha256 = "13xf3685kir4p159hsxrqkj9p2lwgfp0n13h9zadslrd44l8b8j8";
+    rev = "20210314-114017-04b7cedd";
+    sha256 = "sha256-EwoJLwOgoXtTEBbf/4pM+pCCG8fGkVruHVYh2HivCd0=";
     fetchSubmodules = true;
   };
-  cargoSha256 = "1ghjpyd3f5dqi6bblr6d2lihdschpyj5djfd1600hvb41x75lmhx";
+  cargoSha256 = "sha256-OHbWgnlul9VfbPcMdzbuRJG59+myiukkzmnWohj5v2k=";
 
   nativeBuildInputs = [
     pkg-config
@@ -77,21 +81,17 @@ rustPlatform.buildRustPackage {
 
   buildInputs = runtimeDeps;
 
-  installPhase = "" + lib.optionalString stdenv.isLinux ''
+  preFixup = "" + lib.optionalString stdenv.isLinux ''
     for artifact in wezterm wezterm-gui wezterm-mux-server strip-ansi-escapes; do
-      patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $releaseDir/$artifact
-      install -D $releaseDir/$artifact -t $out/bin
+      patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $out/bin/$artifact
     done
   '' + lib.optionalString stdenv.isDarwin ''
-  mkdir -p "$out/Applications"
-  OUT_APP="$out/Applications/WezTerm.app"
-  cp -r assets/macos/WezTerm.app "$OUT_APP"
-  rm $OUT_APP/*.dylib
-  cp -r assets/shell-integration/* "$OUT_APP"
-  cp $releaseDir/wezterm "$OUT_APP"
-  cp $releaseDir/wezterm-mux-server "$OUT_APP"
-  cp $releaseDir/wezterm-gui "$OUT_APP"
-  cp $releaseDir/strip-ansi-escapes "$OUT_APP"
+    mkdir -p "$out/Applications"
+    OUT_APP="$out/Applications/WezTerm.app"
+    cp -r assets/macos/WezTerm.app "$OUT_APP"
+    rm $OUT_APP/*.dylib
+    cp -r assets/shell-integration/* "$OUT_APP"
+    ln -s $out/bin/{wezterm,wezterm-mux-server,wezterm-gui,strip-ansi-escapes} "$OUT_APP"
   '';
 
   # prevent further changes to the RPATH

--- a/pkgs/applications/terminal-emulators/wezterm/default.nix
+++ b/pkgs/applications/terminal-emulators/wezterm/default.nix
@@ -2,19 +2,11 @@
 , rustPlatform
 , lib
 , fetchFromGitHub
-
 , pkg-config
 , fontconfig
 , python3
 , openssl
 , perl
-
-# Apple frameworks
-, CoreGraphics
-, Cocoa
-, Foundation
-, libiconv
-
 , dbus
 , libX11
 , xcbutil
@@ -30,6 +22,11 @@
 , libGL
 , freetype
 , zlib
+# Apple frameworks
+, CoreGraphics
+, Cocoa
+, Foundation
+, libiconv
 }:
 let
   runtimeDeps = [
@@ -50,38 +47,38 @@ let
     wayland
     libGLU
     libGL
+    openssl
   ] ++ lib.optionals (stdenv.isDarwin) [
     Foundation
     CoreGraphics
     Cocoa
     libiconv
   ];
-  pname = "wezterm";
 in
 
-rustPlatform.buildRustPackage {
-  inherit pname;
-  version = "20210314";
+rustPlatform.buildRustPackage rec {
+  pname = "wezterm";
+  version = "20210407-nightly";
 
   src = fetchFromGitHub {
     owner = "wez";
     repo = pname;
-    rev = "20210314-114017-04b7cedd";
-    sha256 = "sha256-EwoJLwOgoXtTEBbf/4pM+pCCG8fGkVruHVYh2HivCd0=";
+    rev = "d2419fb99e567e3b260980694cc840a1a3b86922";
+    sha256 = "4tVjrdDlrDPKzcbTYK9vRlzfC9tfvkD+D0aN19A8RWE=";
     fetchSubmodules = true;
   };
-  cargoSha256 = "sha256-OHbWgnlul9VfbPcMdzbuRJG59+myiukkzmnWohj5v2k=";
+
+  cargoSha256 = "sha256-UaXeeuRuQk+CWF936mEAaWTjZuTSRPmxbQ/9E2oZIqg=";
 
   nativeBuildInputs = [
     pkg-config
     python3
-    openssl.dev
     perl
   ];
 
   buildInputs = runtimeDeps;
 
-  preFixup = "" + lib.optionalString stdenv.isLinux ''
+  preFixup = lib.optionalString stdenv.isLinux ''
     for artifact in wezterm wezterm-gui wezterm-mux-server strip-ansi-escapes; do
       patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $out/bin/$artifact
     done
@@ -101,7 +98,7 @@ rustPlatform.buildRustPackage {
     description = "A GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust";
     homepage = "https://wezfurlong.org/wezterm";
     license = licenses.mit;
-    maintainers = with maintainers; [ steveej ];
+    maintainers = with maintainers; [ steveej SuperSandro2000 ];
     platforms = platforms.unix;
   };
 }

